### ISSUE TITLE
Replace Google Geolocation API with more accurate GPS location from usb GPS module or from the drone

### ISSUE THREAD

**Issue by mdgilene** [mdgilene/trash-scout#1]
The current implementation makes use of Google's Geolocation API. This is too inaccurate due to only having the IP address of the base station in order to locate the machine.

- [ ]  Replace Geolocation API
- [ ]  Remove node-macaddress dependency

**Comment by m-cameron** [comment]
Right, you can't use Google API, IP geolocation API or wifi based API inside drone.

You should consider the drone internal GPS module.
